Sources/Noora/Components/TextPrompt.swift

```diff
@@ -10,6 +11,7 @@ struct TextPrompt {
     let collapseOnAnswer: Bool
     let renderer: Rendering
     let standardPipelines: StandardPipelines
+    let logger: Logger?
```

Comment:
I'd log in here:
- What the prompt has been
- What answer we received from the developer